evaluator/perception_online_evaluator/src/metrics_calculator.cpp

```diff
@@ -399,6 +402,35 @@ MetricStatMap MetricsCalculator::calcYawRateMetrics(const ClassObjectsMap & clas
   return metric_stat_map;
 }
 
+MetricStatMap MetricsCalculator::calcObjectsCountMetrics() const
+{
+  MetricStatMap metric_stat_map;
+
+  for (const auto & [label, count] : historical_detection_count_map_) {
+    Stat<double> stat;
+    stat.add(
+      static_cast<double>(static_cast<double>(count) / static_cast<double>(objects_count_frame_)));
```

Comment:
historical objects count is calculated by being divided by `objects_count_frame_` , so it means the count in cycle not total?

Reply:
the answer is yes i guess
I changed in this way to eliminate the influence of the execution times.